### DISCUSSION TITLE
Displaying Karamata notation of Stirling numbers

### DISCUSSION THREAD

**Discussion by matthias-buttgereit** [typst/typst#1605] [Q&A] [answered]
Is there a way/hack to display [stirling numbers](https://en.wikipedia.org/wiki/Stirling_number) in their Karamata notation?

The notation basically looks like a binomial coefficient but with square brackets for the first kind or curly brackets for the second kind of Stirling numbers.

**Comment by Enivex** [comment, marked as answer]
You could use a vector with custom delimiters

**Reply by matthias-buttgereit** [reply]
That would do it, thank you!
